app/models/feedback_message.rb

```diff
@@ -8,6 +8,8 @@ class FeedbackMessage < ApplicationRecord
 
   validates :feedback_type, :message, presence: true
   validates :reported_url, :category, presence: { if: :abuse_report? }
+  validates :reported_url, length: { maximum: 50 }
```

Comment:
I believe that this could be rewritten to all be on a single line like so: 
`validates :reported_url, :category, presence: { if: :abuse_report? }, length: { maximum: 50 } ` 😄

Reply:
Good catch, thanks!